src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -74,11 +74,13 @@ import {
   unemployabilityFormIntro,
 } from '../pages';
 
-import { PTSD } from '../constants';
+import { createFormConfig781, createFormConfig781a } from './781';
+
+import { PTSD, PTSD_INCIDENT_ITERATION } from '../constants';
 
 import fullSchema from './schema';
 
-const formConfig = {
+const createFormConfig = {
```

Comment:
Was this an errant 'replace all'? Otherwise, not sure why this changed.

Reply:
Oh man good catch... Definitely missed that. Fixed.